test/e2e-common.sh

```diff
@@ -32,9 +32,15 @@ function add_sidecar_registry() {
     rm -f ${TMPF}.read
 }
 
-# Add the git_clone task
-function add_git_clone_task() {
-    kubectl -n ${tns} apply -f ./task/git-clone/0.1/git-clone.yaml
+function add_task() {
+    local array version task
+    task=${1}
+    version=${2}
+    [[ "${version}" == "latest" ]] && {
+        array=($(echo task/${task}/*|sort -u))
```

Comment:
should we handle the scenario of wrong task name

Reply:
why not, sure but it will fail anyway since we run under -e :-)